# Copyright Contributors to the OpenVDB Project SPDX-License-Identifier: MPL-2.0

  This Source Code Form is subject to the terms of the Mozilla Public
  License, v. 2.0. If a copy of the MPL was not distributed with this
  file, You can obtain one at http://mozilla.org/MPL/2.0/.
----
unchanged:
--- nanovdb.orig/examples/CMakeLists.txt	2021-03-30 18:32:35.319835787 -0700
+++ nanovdb/examples/CMakeLists.txt	2021-03-30 18:35:33.854775052 -0700
@@ -67,7 +67,7 @@ function(_ADD_EXAMPLE)
             "-D${NANOVDB_USE_TBB_FLAG}" "-D${NANOVDB_USE_BLOSC_FLAG}"
             "-D${NANOVDB_USE_ZLIB_FLAG}")
 
-  install(TARGETS ${EXAMPLE_NAME} DESTINATION examples)
+  install(TARGETS ${EXAMPLE_NAME} DESTINATION share/nanovdb/examples)
 
 endfunction()
 
@@ -137,7 +137,7 @@ function(_ADD_EXAMPLE_OPENVDB)
             "-D${NANOVDB_USE_TBB_FLAG}" "-D${NANOVDB_USE_BLOSC_FLAG}"
             "-D${NANOVDB_USE_ZLIB_FLAG}")
 
-  install(TARGETS ${EXAMPLE_NAME} DESTINATION examples)
+  install(TARGETS ${EXAMPLE_NAME} DESTINATION share/nanovdb/examples)
 
 endfunction()
 
only in patch2:
unchanged:
--- nanovdb.orig/examples/benchmark/CMakeLists.txt	2021-03-30 18:47:07.353730868 -0700
+++ nanovdb/examples/benchmark/CMakeLists.txt	2021-03-30 18:47:45.623218419 -0700
@@ -50,6 +50,6 @@ target_compile_definitions(
           "-D${NANOVDB_USE_TBB_FLAG}" "-D${NANOVDB_USE_BLOSC_FLAG}"
           "-D${NANOVDB_USE_ZLIB_FLAG}")
 
-install(TARGETS benchmark DESTINATION examples)
+install(TARGETS benchmark DESTINATION share/examples)
 
 # ---------------------------------------------------------------
